proto/public_key.proto

```diff
@@ -13,24 +13,42 @@ service PublicKeyService {
     rpc Add(PublicKeyRequest) returns (PublicKeyResponse) {};
 }
 
+message HeaderAuth {
+    string header = 1;
+    string value = 2;
+}
+
 message PublicKeyRequest {
```

Comment:
I guess we can just stop sending that singing key along, shouldn't break anything if this gets deployed first though since that field would just be ignored anyways I suppose

Reply:
👍  I wasn't sure if the response is parsing the signing public key and wanting it to always exist.